doc/index.md

```diff
@@ -23,14 +23,20 @@ See [Security](security.md) for detailed information.
 ```
 ````
 
-## Support
+## Project and community
 
-See {doc}`support` for information on how to get help.
+LXD is free software and developed under the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0).
+It’s an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
 
-## Contributing
+The LXD project is sponsored by [Canonical Ltd](https://www.canonical.com).
 
-Fixes and new features are greatly appreciated.
-See {doc}`contributing` for more information.
+- [Code of Conduct](https://github.com/lxc/lxd/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
Can we get inclusive naming check to ignore URLs?

Reply:
Not in general, unfortunately, only through exceptions. :(